frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -393,6 +393,7 @@ const SECTIONS = updateSectionsWithPlugins({
         widget: () => {
           return (
             <Breadcrumbs
+              size="large"
```

Comment:
Make the breadcrumb size bigger to match [the design](https://www.figma.com/file/SAi6pRHuhrWeVRuOHH4evg/Better-communicate-embedding-options-in-Admin-settings?node-id=83%3A618).